ILRepack.Tests/Utils/BamlDocumentMatcher.cs

```diff
@@ -153,7 +153,23 @@ private bool Equals(IEnumerable expected, IEnumerable actual)
 
         private List<BamlRecord> GetRelevantRecords(BamlDocument document)
         {
-            return document.Where(node => !(node is LineNumberAndPositionRecord || node is LinePositionRecord)).ToList();
+            var list = document.Where(node => !(node is LineNumberAndPositionRecord || node is LinePositionRecord)).ToList();
```

Comment:
Those checks could be merged with the AssemblyInfoRecord checks.

Reply:
I'm not sure I understand your idea here.

Reply:
Actually, all good, I missed the fact we put back the rest at line169.